plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -61,6 +63,78 @@ public void testDescribeTable()
         assertEquals(actualColumns, expectedColumns);
     }
 
+    @Test
+    public void createTableWithEveryType()
```

Comment:
nit: would it take much more test-execution time to provide type and expected type via test parameters and use `@DataProvider` to pass those. Then each test execution would create table with just one column. Having separate tests for each type can make investigating failures later on easier.